specs/Modal.spec.js

```diff
@@ -92,6 +92,12 @@ describe('Modal', function () {
     });
   });
 
+  it('handles case when child has no tabbable elements', function() {
```

Comment:
That's intentional. The spec simply verifies that an exception is not thrown when a tab is pressed. Without the bugfix, the spec will fail.